tests/Annotation/EntityTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: szymon
+ * Date: 05.10.17
+ * Time: 13:26
+ */
+
+namespace Opstalent\CrudBundle\Tests\Annotation;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use Opstalent\CrudBundle\Tests\Resources\Entity\CategoryEntityMock;
+use Opstalent\CrudBundle\Tests\Resources\Entity\CategoryExceptionEntityMock;
+
+/**
+ * Class EntityTest
+ * @package Opstalent\CrudBundle
+ */
+class EntityTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @group EntityAnnotation
+     * @group Annotation
+     * @test
+     **/
+    public function checkForExistingAction()
+    {
+        $reader = new AnnotationReader();
+        $metadata = $reader->getClassAnnotation(
+            new \ReflectionClass(CategoryEntityMock::class),
+            "Opstalent\\CrudBundle\\Annotation\\Entity"
+        );
+        $this->assertNotNull($metadata);
+        $this->assertNotNull($metadata->actions);
+    }
+
+    /**
+     * @group EntityAnnotation
+     * @group Annotation
+     * @test
+     * @expectedException
```

Comment:
what type of exception? also use [this](https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/TestCase.php#L576)